csrc/transform_replay.h

```diff
@@ -158,12 +165,14 @@ class TORCH_CUDA_CU_API TransformReplay {
       const TensorView* consumer,
       const TensorView* producer,
       int64_t producer_compute_at_axis,
-      bool replay_swizzle = false);
+      bool replay_swizzle = false,
+      bool replay_allocation = false);
   static std::pair<TensorDomain*, size_t> replayCasP(
       const TensorView* consumer,
       const TensorView* producer,
       int64_t producer_compute_at_axis,
       const RootDomainMap& root_map,
+      bool replay_allocation = false,
```

Comment:
And indeed this seems like a bug as the order of `replay_allocation` and `replay_swizzle` is swapped in the function definition.

```
std::pair<TensorDomain*, size_t> TransformReplay::replayCasP(
    const TensorView* consumer,
    const TensorView* producer,
    int64_t producer_pos,
    const RootDomainMap& root_map,
    bool replay_swizzle,
    bool replay_allocation) {
```

Reply:
This definitely is, "fortunately" it is only in this header file, where the name is just ignored by the compiler, and in the cpp file, I am using the correct order.😅

Reply:
After cleanup, this kind of bug is no longer possible.